doc/release-notes.md

```diff
@@ -0,0 +1,97 @@
+# Changes to `ga4gh/schemas` `master` branch since version `v0.5.1` (Oct 2, 2014)
+
+### Pervasive changes
+
+Renamed all protocols and records to remove the "`GA`" prefix from
+their names.  (Hence `GASearchVariantSetsRequest` is now
+`SearchVariantSetsRequest`, etc.)  The exception is `GAException`,
+which is unchanged.
+
+There is no longer one unitary namespace.  Objects now reside in
+"`org.ga4gh.models`", methods in "`org.ga4gh.methods`".
+
+Updated the version number to `0.6.0-SNAPSHOT`.
+
+### Added
+
+Datasets are now used throughout the API.
+
+#### Methods:
+
+* `getDataset`
+* `searchDatasets`
+* `getReadGroupSet`
+* `getReadGroup`
+* `getVariant`
+* `getVariantSet`
+* `getCallSet`
+
+#### Records:
+
+* `ExternalIdentifier`
+* `Experiment`
+* `Dataset`
+* `ReadStats`
+* `Fragment` _(apparently unused)_
```

Comment:
The `Fragment` should really be removed before we make the release. It currently doesn't do anything, so we should just remove it from the schema entirely. It'll be easy enough to add back in later when we figure out what we need it for.